Source/core/ResourceMonitor.h

```diff
@@ -154,18 +155,18 @@ namespace Core {
 
                 _adminLock.Unlock();
 
-                delete _monitor;
+                _monitor.reset();
```

Comment:
not strictly necessary -- the line could be removed entirely because we are about to get destroyed anyway